src/site/twitch.tv/modules/media-player/MediaPlayer.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+	<template v-if="shouldShowVideoStats">
+		<Teleport v-if="doTeleport" :to="teleportLocation">
+			<div
+				id="seventv-video-stats"
+				ref="videoStatsRef"
+				@mouseenter="videoStatsTooltip.show(videoStatsRef)"
+				@mouseleave="videoStatsTooltip.hide()"
+			>
+				<figure v-if="videoStats.playbackRate > 1"><ForwardIcon /></figure>
+				<figure v-else><GaugeIcon /></figure>
+				<span>{{ videoStats.liveLatency }}s</span>
+			</div>
+		</Teleport>
+	</template>
+</template>
+
+<script setup lang="ts">
+import { onUnmounted, ref, toRef, watch } from "vue";
+import { isNumber } from "@vueuse/core";
+import { HookedInstance } from "@/common/ReactHooks";
+import {
+	defineNamedEventHandler,
+	definePropertyHook,
+	unsetNamedEventHandler,
+	unsetPropertyHook,
+} from "@/common/Reflection";
+import { useConfig } from "@/composable/useSettings";
+import { useTooltip } from "@/composable/useTooltip";
+import ForwardIcon from "@/assets/svg/icons/ForwardIcon.vue";
+import GaugeIcon from "@/assets/svg/icons/GaugeIcon.vue";
+import VideoStatsTooltip from "./VideoStatsTooltip.vue";
+
+const props = defineProps<{
+	instance: HookedInstance<Twitch.MediaPlayerComponent>;
+	videoStatsTeleportLocation: HTMLElement | undefined;
+}>();
+
+const teleportLocation = toRef(props, "videoStatsTeleportLocation");
+const doTeleport = ref<boolean>(false);
+
+const shouldShowVideoStats = useConfig<boolean>("player.video_stats");
+const actionOnClick = useConfig<number>("player.action_onclick");
+
+const mediaPlayerProps = ref<Twitch.MediaPlayerComponent["props"]>();
+// const statePath = ref<string>();
+const videoStats = ref({
+	liveLatency: "-.--",
+	droppedFrames: 0,
+	playbackRate: 0,
+	bitrate: "0",
+	width: 0,
+	height: 0,
+	framerate: 0,
+});
+
+if (props.instance.component.props) {
```

Comment:
Unnecessary `if` block, `props.instance.component` should never be nullish as it is declared a required prop.